forms/static/forms/admin/dependent_fields.js

```diff
@@ -0,0 +1,77 @@
+(function($){
+    const special_tabs = function () {
+        // For TV show the age
+        if($('#televisionsheet_form').length > 0){
+            $('.field-age').show();
+        }
+        // For Internet show the age
+        if($('#internetnewssheet_form').length > 0){
+            $('.field-age').show();
+        }
+    };
+
+    const long_monitoring_mode = function () {
+        $('.field-page_number').show();
+        $('.field-space').show();
+        $('.field-age').show();
+        $('.field-is_photograph').show();
+        $('.field-special_qn_1').show();
+        $('.field-special_qn_2').show();
+        $('.field-special_qn_3').show();
+        $('.field-item_number').show();
+        $('.field-webpage_layer_no').show();
+        $('.field-victim_or_survivor').show();
```

Comment:
1. We already have `$('.field-age')` on L16 but we're again searching for it on L5/L9, why don't we just pass it to `special_tabs` function?,
2. Does `long_monitoring_mode` need to call `special_tabs`? looks like it already does `$('.field-age').show()`, and
3. What's so special about `special_tabs` :sunglasses:

Reply:
Sure I'll update this.
For 3, it's because they're the only tabs that show `field-age` on short monitoring mode.

Reply:
3. was more of a joke about the name than actual something that needs to be addressed @esirK :grin: